core/build.gradle

```diff
@@ -178,6 +178,8 @@ dependencyManagement {
     dependency group: 'org.apache.commons',                 name: 'commons-collections4',        version: commonsCollections4Version
     dependency group: 'org.apache.commons',                 name: 'commons-lang3',               version: apacheCommonsLangVersion
     dependency group: 'com.cloudera.cdp',                   name: 'cdp-sdk-java',                version: cdpSdkVersion
+    dependency group: 'org.apache.hadoop',                  name: 'hadoop-yarn-api',           version: hadoopVersion
```

Comment:
`hadoop-yarn-api` and `consul-api` are added as dependencies of `autoscale` package.